resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
The web app has lots of edge cases which are tricky to decide whether they're pages or not.  Technically, the user only every loads a single page and everything else is javascript.  So we had to make many arbitrary decisions about which interactions counted as a new page (eg is opening an overlay a separate page?  What if that overylay takes up the entire screen? etc etc)